src/frame/content.rs

```diff
@@ -109,6 +111,10 @@ impl Content {
             Self::TableOfContents(table_of_contents) => {
                 Comparable(vec![Cow::Borrowed(table_of_contents.element_id.as_bytes())])
             }
+            Self::UFID(ufid) => Comparable(vec![
+                Cow::Borrowed(ufid.owner_identifier.as_bytes()),
+                Cow::Borrowed(ufid.identifier.as_slice()),
```

Comment:
This `unique` function implements the uniqueness as defined by the ID3 spec. Given the spec:

    There are allowed to be more than one UFID frame per tag, but they must have unique owner identifiers

Then only the owner_identifier field should be returned here